.github/workflows/main.yml

```diff
@@ -8,11 +8,11 @@ on:
   push:
```

Comment:
> It seems that the login/logout behavior isn't considered properly. Let's say I had some products in my cart, now I log in, and I expect product cards to show the cart quantity from my cart, but this doesn't seem to be the case.
And vice versa for the log out: after I click "log out" product cards still show quantity from the cart

Nice catch, I'ved push a fix on this:


[Screencast from 09-01-2025 03:19:14 PM.webm](https://github.com/user-attachments/assets/9e13efff-b9f2-466f-ac2c-4600584e0af2)